Gems/EMotionFX/Code/Include/Integration/AnimGraphNetworkingBus.h

```diff
@@ -36,6 +36,8 @@ namespace EMotionFX
         virtual void UpdateActorExternal(float deltatime) = 0;
         virtual void SetNetworkRandomSeed(AZ::u64 seed) = 0;
         virtual AZ::u64 GetNetworkRandomSeed() const = 0;
+        virtual void SetActorThreadIndex(AZ::u32 threadIndex) = 0;
```

Comment:
Is this used in this PR? I don't see where.

Reply:
It's used in `NetworkAnimationComponent` which is in MP Sample, so has to be a separate PR..
Seems all of these functions are used only in that component

Reply:
https://github.com/o3de/o3de-multiplayersample/pull/212